services/crontab.py

```diff
@@ -1,36 +1,42 @@
+from django.conf import settings
 from crontab import CronTab
 from cron import models
 from django.db.models.signals import post_save
 from django.dispatch import receiver
 from cron.models import Job
+import os
+import sys
+
 
 class CrontabService:
-  def __init__(self):
-    self.cron = CronTab(user=True)
-  
-  def _get_curl_command(self, job):
-    return f'request_curl --url {job.fullURL} --token {job.application.token} --method {job.method} &>> /var/log/poker.logs'
-
-  def createJob(self, dbJob):
-    job = self.cron.new(command = self._get_curl_command(dbJob), comment=str(dbJob.id))
-    job.setall(dbJob.timeExpression)
-    self.cron.write()
-  
-  def removeJob(self, dbJob):
-    job = self.cron.find_comment(str(dbJob.id))
-    self.cron.remove(job)
-    self.cron.write()
-
-  def refreshAllJob(self):
-    self.cron.remove_all()
-    self.cron.write()
-    for job in models.Job.objects.all():
-      self.createJob(job)
+    def __init__(self):
+        self.cron = CronTab(user=True)
+
+    def _get_curl_command(self, job):
+        return f'{sys.executable} {os.path.join(settings.BASE_DIR, "manage.py")} sendrequest --url {job.fullURL} --token {job.application.token} --method {job.method} &>> /var/log/poker.logs'
```

Comment:
change this command to only send the job_id and don't redirect the output

Reply:
done